lib/bugsnag/railtie.rb

```diff
@@ -51,7 +49,7 @@ class Railtie < Rails::Railtie
         ActiveRecord::Base.send(:include, Bugsnag::Rails::ActiveRecordRescue)
       end
 
-      Bugsnag.configuration.app_type = "rails" unless Bugsnag::Railtie.running_as_dependency
+      Bugsnag.configuration.app_type = "rails"
```

Comment:
This change relies on the fact that in the other cases (sidekiq etc) we know that we are running in that env. This is definitely a little broad, but I think its OK. This will ultimately be set to rails in the console and in the server case, which is what we want.

It is also initially set to rails in other apps (for example in a sidekiq app that also has rails) but because we later know we are running in a sidekiq app, we change the app_type then to be sidekiq.